docs/installation.rst

```diff
@@ -223,14 +208,11 @@ the libev event loop by doing the following:
 
 .. code-block:: python
 
-    >>> from cassandra.io.libevreactor import LibevConnection
-    >>> from cassandra.cluster import Cluster
+    >>> from scylla.io.libevreactor import LibevConnection
+    >>> from scylla.cluster import Cluster
 
     >>> cluster = Cluster()
     >>> cluster.connection_class = LibevConnection
     >>> session = cluster.connect()
 
-(*Optional*) Configuring SSL
```

Comment:
why removing this ?

I guess it's a bit old, and might be out of date.
but it might be good reference on how to configure SSL on the the driver
and I hate to remove something is clearly a community attribution

maybe we should point to other parts of the docs:
https://scylladb.github.io/python-driver/security/#ssl